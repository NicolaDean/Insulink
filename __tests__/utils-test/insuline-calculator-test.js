import InsulineCalculator from "../../src/utils/insulineCalculator";

test('Test totalDose', () => {
    const ic = new InsulineCalculator(50,1,80);

    const res = ic.totalDose(44,230);
    expect(res).toMatchSnapshot();
});