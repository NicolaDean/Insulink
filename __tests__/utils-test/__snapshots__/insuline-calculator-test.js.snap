// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test totalDose 1`] = `
String {
  "0": "4",
}
`;
